scripts/billing/sql/forecast_net_balance.sql

```diff
@@ -0,0 +1,114 @@
+/*
+Query returns the forecasted future balance for the next month given
+historical usage trends
+*/
+
+SELECT
+  month,
+  balance,
+  future_balance
+FROM (
```

Comment:
Sub-selects parens should be placed on a new line and indented by 2-spaces ([http://go/sqlguide#clause-block](http://go/sqlguide#clause-block))

Please also apply to all the sub-selects below as well. If heavily nested, think about using the `WITH` clause for greater readability.